consumer/src/events.rs

```diff
@@ -518,14 +555,14 @@ impl Processor {
         } = tx.addresses;
         let id = key.id.parse()?;
 
-        let collection = collections::Model {
-            id,
-            master_edition: master_edition.to_string(),
-            owner: owner.to_string(),
-            metadata: metadata.to_string(),
-            associated_token_account: associated_token_account.to_string(),
-            mint: mint.to_string(),
-            update_authority: update_authority.to_string(),
+        let collection = collections::ActiveModel {
+            id: Set(id),
+            master_edition: Set(master_edition.to_string()),
+            owner: Set(owner.to_string()),
+            metadata: Set(metadata.to_string()),
+            associated_token_account: Set(associated_token_account.to_string()),
+            mint: Set(mint.to_string()),
+            update_authority: Set(update_authority.to_string()),
             ..Default::default()
         };
```

Comment:
The create function converts the model to an active model.

Reply:
I  was using default generated id before. 
FYI Default values for model and active model are different. Example, default id will be zero uuid for model but active model is going to set at db side.